fastapi/security/oauth2.py

```diff
@@ -163,6 +163,45 @@ def __init__(
         return param
 
 
+class OAuth2AuthorizationCodeBearer(OAuth2):
+    def __init__(
+        self,
+        authorizationUrl: str,
+        tokenUrl: str,
+        refreshUrl: str = None,
+        scopes: dict = None,
+        scheme_name: str = None,
+        auto_error: bool = True,
+    ):
+        if not scopes:
+            scopes = {}
+        flows = OAuthFlowsModel(
+            authorizationCode={
+                "authorizationUrl": authorizationUrl,
+                "tokenUrl": tokenUrl,
+                "refreshUrl": refreshUrl,
+                "scopes": scopes,
+            })
+        super().__init__(
+            flows=flows, scheme_name=scheme_name, auto_error=auto_error
+        )
+
+    async def __call__(self, request: Request) -> Optional[str]:
+        authorization: str = request.headers.get("Authorization")
+        print(authorization)
```

Comment:
Presumably you didn't mean to keep this print statement there?

Reply:
Thanks, just removed it.